docs/_partials/_using-multiple-promises.md

```diff
@@ -264,13 +258,27 @@ To recap the steps we took:
 
 This is only the beginning of working with Promises. Next you will learn how to write and update Promises, and in the final thoughts we will showcase the composability of Promises to further optimise this workflow from three requests down to one.
 
-## Tearing it all down {#teardown}
+## Cleanup environment {#cleanup}
 
-The next section in this tutorial requires a clean Kratix installation. Before heading to it, please clean up your environment by running:
+To clean up your environment you need to delete the Promises and the Resource Requests.
+Kratix will automatically cleanup any Resource Requests when the parent Promise is deleted.
 
+To delete all the promises
 ```bash
-kind delete clusters platform worker
+kubectl --context kind-platform delete promises --all
+```
+
+Verify all the promises are deleted
+```console
+kubectl --context kind-platform get promises
 ```
+
+Verify all the resources created by the Promise and Resource requests are gone (this might take a couple minutes)
```

Comment:
```suggestion
When Resource Requests are deleted, it may take a few minutes for the running infrastructure to be removed. When this is complete, you the worker cluster should not have any pods or knative specific namespaces:
```